src/util/entities/Member.ts

```diff
@@ -260,9 +260,9 @@ export class Member extends BaseClassWithoutId {
 					},
 				},
 			}),
-			await Role.findOneOrFail({ where: { id: role_id, guild_id } }),
```

Comment:
I'm in bed currently, but why was the await removed? This function throws if the role couldn't be found